src/pages/AddReservations.jsx

```diff
@@ -0,0 +1,263 @@
+/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { useAuthHeader } from 'react-auth-kit';
+import axios from 'axios';
+import Alert from '../layouts/Alert';
+
+function AddReservations() {
```

Comment:
- [ ] I can't make reservations from details pages:

If I go to a details page (after reloading the page), and try to reserve a car from there, this happens:

![image](https://github.com/AJ193/Final_Capstone_Front_End/assets/57421823/1cfcbd72-9714-40b0-9b6b-d32e48ddd92d)

The server won't allow me:


![image](https://github.com/AJ193/Final_Capstone_Front_End/assets/57421823/6ebf79dc-68de-4d35-9df3-addf2e02c955)

This one won't work even if I reload the page. 



Kindly, fix this bug so your app works as intended. 👍 

_note: The reservations made from the  "Add reservation" link are working well._